utils/utils.jsx

```diff
@@ -249,6 +249,16 @@ export function getTimestamp() {
     return Date.now();
 }
 
+export function getRemainingDaysFromFutureTimestamp(timestamp) {
+    var oneDay = 24 * 60 * 60 * 1000; // hours*minutes*seconds*milliseconds
+    var today = new Date().getTime();
+    if (today >= timestamp) {
+        return -1;
+    }
+
+    return Math.round(Math.abs((timestamp - today) / (oneDay)));
+}
+
```

Comment:
I think you can do something like
```
date = new Date();
date.setUTCDate(date.getDate() -1);
return date;
```
I tend to prefer leaving date handling to the built in functions

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCDate